examples/03_connectivity/plot_multi_subject_connectome.py

```diff
@@ -26,59 +25,81 @@ def plot_matrices(cov, prec, title, labels):
     span = max(abs(prec.min()), abs(prec.max()))
 
     # Display covariance matrix
-    plotting.plot_matrix(cov, cmap=plotting.cm.bwr,
-                         vmin=-1, vmax=1, title="%s / covariance" % title,
-                         labels=labels)
+    plotting.plot_matrix(
+        cov,
+        cmap=plotting.cm.bwr,
+        vmin=-1,
+        vmax=1,
+        title=f"{title} / covariance",
+        labels=labels,
+    )
     # Display precision matrix
-    plotting.plot_matrix(prec, cmap=plotting.cm.bwr,
-                         vmin=-span, vmax=span, title="%s / precision" % title,
-                         labels=labels)
+    plotting.plot_matrix(
+        prec,
+        cmap=plotting.cm.bwr,
+        vmin=-span,
+        vmax=span,
+        title=f"{title} / precision",
+        labels=labels,
+    )
 
 
 ##############################################################################
 # Fetching datasets
 # ------------------
 from nilearn import datasets
+
 msdl_atlas_dataset = datasets.fetch_atlas_msdl()
 rest_dataset = datasets.fetch_development_fmri(n_subjects=n_subjects)
 
 # print basic information on the dataset
-print('First subject functional nifti image (4D) is at: %s' %
-      rest_dataset.func[0])  # 4D data
+print(
+    f"First subject functional nifti image (4D) is at: {rest_dataset.func[0]}"
+)
 
 
+# A "memory" to avoid recomputation
+from joblib import Memory
```

Comment:
I think we can remove the associated comment and move the import after the the "Extracting region signals" title below.

Reply:
Actually along with your other comment, it seems that this is not actually used anywhere so I removed it.

Unless I am missing something on how this caching is used.